src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -1452,6 +1430,27 @@ void FillEmitter::emit_isa(const std::vector<size_t> &in, const std::vector<size
     Vmm src_vmm = Vmm(in[0]);
     Vmm dst_vmm = Vmm(out[0]);
 
+    if (is_full_reg())
+        fill_full<Vmm>(dst_vmm);
+    else
+        fill_tail<Vmm>(src_vmm, dst_vmm);
+}
+
+template <typename Vmm>
+void FillEmitter::fill_full(const Vmm& dst_vmm) const {
+    // Optimized impl for zero
+    if (fill_value == int32_t(0x0)) {
+        h->uni_vpxor(dst_vmm, dst_vmm, dst_vmm);
+    } else {
+        Xmm xmm = Xmm(dst_vmm.getIdx());
+        h->mov(Reg64(aux_gpr_idxs[0]), fill_value);
+        h->vmovq(xmm, Reg64(aux_gpr_idxs[0]));
+        h->vbroadcastss(dst_vmm, xmm);
```

Comment:
Broadcast can also read from memory, can't it?
If so, are you sure it's more efficient to perform `mov` + `vmovq` instead of reading from table as you do in `fill_tail`?